### PR TITLE
Run and save nexmark benchmark results as part of CI.

### PR DESCRIPTION
**Pull request by gz**

### CODE REVIEW

**Comment by gz** [comment]
Windows runner is running out of disk-space:

```
   Compiling dbsp v0.1.0 (D:\a\database-stream-processor\database-stream-processor)
LLVM ERROR: IO failure on output stream: no space on device
error: could not compile `dbsp`
warning: build failed, waiting for other jobs to finish...
LLVM ERROR: IO failure on output stream: no space on device
error: could not compile `dbsp`
Error: The process 'C:\Users\runneradmin\.cargo\bin\cargo.exe' failed with exit code 101
```

File: scripts/ci.bash [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#
+set -ex
+
+# Clean-up old results
+rm -rf gh-pages
+NEXMARK_CSV_FILE='nexmark_results.csv'
+rm -f ${NEXMARK_CSV_FILE}
+
+# Run benchmarks
+cargo bench --bench nexmark --features with-nexmark -- --first-event-rate=1000000 --max-events=100000 --cpu-cores 8  --num-event-generators 6 --source-buffer-size 10000 --input-batch-size 40000 --csv
```
Comment (ryzhyk) [inline scripts/ci.bash:13]: This is without persistence, right?  In that case, we want `--first-event-rate=10000000` (10M).

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -38,11 +38,11 @@ jobs:
               os: [self-hosted, Linux, skylake-2x],
             }
           - { rust: nightly, target: x86_64-apple-darwin, os: macos-latest }
-          - {
-              rust: nightly,
-              target: x86_64-pc-windows-msvc,
-              os: windows-latest,
-            }
+          #- {
```
Comment (ryzhyk) [inline .github/workflows/main.yml:41]: Giving up on windows completely is a bit too radical.  Perhaps we could just build/test with minimal features?

File: scripts/ci.bash (hunk 1 above)
Comment (gz) [reply scripts/ci.bash:13]: Yes, I'll increase it, left it low for now so it completes fast

File: .github/workflows/main.yml (hunk 2 above)
Comment (gz) [reply .github/workflows/main.yml:41]: It runs out of disk-space during build -- I'll try to build with less features/dependencies.

Comment (ryzhyk) [reply .github/workflows/main.yml:41]: Even running with no features would be fairly useful, compared to dropping windows altogether.

File: scripts/ci.bash (hunk 1 above)
Comment (gz) [reply scripts/ci.bash]: Fixed

File: .github/workflows/main.yml (hunk 2 above)
Comment (gz) [reply .github/workflows/main.yml]: Disabling the CI cache for windows makes it work again with no need to limit features.

**Pull request merged**